lib/r10k/module/base.rb

```diff
@@ -66,6 +66,27 @@ def full_path
     path.to_s
   end
 
+  # Delete the spec dir unless @deploy_spec has been set to true
+  def delete_spec_dir
+    unless @deploy_spec
+      spec_path = @path + 'spec'
```

Comment:
We should probably do a `File.join` here.

Reply:
`File.join` will return a string, while a `<Pathname> + String` operation will return a Pathname object. I could wrap the `File.join` in a new Pathname, but I don't think that is any better...I'm not sure I need a `Pathname` for the `Dir.exist?` and `FileUtils.rm_rf` calls though... 🤷

Reply:
Touche.

I didn't realize that `@path` was a Pathname. In that case you might want to use the instance methods defined on Pathname for symlink?, realpath, directory?...

Reply:
Oh, yeah, that sounds good to me. 👍